apple/RNGestureHandler.h

```diff
@@ -87,4 +89,10 @@
 - (void)sendEvent:(nonnull RNGestureHandlerStateChange *)event;
 - (void)sendTouchEventInState:(RNGestureHandlerState)state forViewWithTag:(nonnull NSNumber *)reactTag;
 
+#if !TARGET_OS_OSX
+- (void)setCurrentPointerType:(nonnull UIEvent *)event;
+#else
+- (void)setCurrentPointerType;
```

Comment:
Can we use a different name for macos that somehow suggests that it will be mouse?

Reply:
Done in [adbdf3c](https://github.com/software-mansion/react-native-gesture-handler/pull/2760/commits/adbdf3c613917eccbecff1b5bb3257b8bb0d0d6d)